packages/components/storybook/config.js

```diff
@@ -14,7 +14,7 @@ addDecorator( withA11y );
 addDecorator( withKnobs );
 configure(
 	[
-		require.context( '../docs', true, /\/.+\.mdx$/ ),
+		// require.context( '../docs', true, /\/.+\.mdx$/ ),
```

Comment:
I didn't investigate what to do about `mdx`. The simplest approach would  load only files with the `.js` extension using:
https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core#storykindrege

Reply:
Yeah, let's do that. Most of these addons don't have support for looking for code snippets inside `.mdx` files unfortunately.

Reply:
I resolved it a bit differently, by loading `MDX` files only in non-test env.

Reply:
Nice!